cmd/test_server/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+	"roci.dev/replicache-client/repm"
+)
+
+func main() {
+	impl(os.Args[1:], os.Stdin, os.Stdout, os.Stderr, os.Exit)
+}
+
+func impl(args []string, in io.Reader, out, errs io.Writer, exit func(int)) {
```

Comment:
The one thing I didn't consider with this approach is that if we ever want push bindings, we'll need to upgrade this to web sockets. Do you see any problem with that?

Reply:
Not sure. Will have to think about it further.